packages/usa-in-page-navigation/src/index.js

```diff
@@ -14,6 +14,7 @@ const IN_PAGE_NAV_TITLE_HEADING_LEVEL = "h4";
 const IN_PAGE_NAV_SCROLL_OFFSET = 0;
 const IN_PAGE_NAV_ROOT_MARGIN = "0px 0px 0px 0px";
 const IN_PAGE_NAV_THRESHOLD = "1";
+const IN_PAGE_NAV_MINIMUM_HEADING_COUNT = 3;
```

Comment:
Updated!

Reply:
**question:** Would we want to consider making the default minimum a system setting? This would enable users to update _all_ in page nav components on their site in one place, and use the data attribute for specific instances. 🤔